ts/screens/modal/IdentificationModal.tsx

```diff
@@ -424,11 +425,9 @@ class IdentificationModal extends React.PureComponent<Props, State> {
           accessible={true}
           ref={this.headerRef}
         >
-          {I18n.t(
-            isValidatingTask
-              ? "identification.titleValidation"
-              : "identification.title"
-          )}
+          {isValidatingTask
+            ? I18n.t("identification.titleValidation")
+            : `${I18n.t("identification.title")} ${this.props.profileName}!`}
```

Comment:
Why don't to add this directly in locales definition?

For Example:
```
title: Ciao {{profileName}}
```

and then in component code:
```
${I18n.t("identification.title", {
profileName: this.props.profileName
})}
```